const newOrder = async (req, res, next) => {
    try {
    }
    catch (error) { }
};
export { newOrder };
